cmd/build/helmify/kustomize-for-helm.yaml

```diff
@@ -89,6 +89,8 @@ spec:
             - --disable-cert-rotation={{ .Values.controllerManager.disableCertRotation }}
             - --max-serving-threads={{ .Values.maxServingThreads }}
             - --tls-min-version={{ .Values.controllerManager.tlsMinVersion }}
+            - --validating-webhook-configuration-name={{ .Values.validatingWebhookName }}
+            - --mutating-webhook-configuration-name={{ .Values.mutatingWebhookName }}
```

Comment:
I think we also need to update the name in the rbac that gets created by kubebuidler thru this tag:

- https://github.com/open-policy-agent/gatekeeper/blob/master/pkg/webhook/mutation.go#L52-L56

There should be some prior art to do this replacement. I think maybe the labels patch is a good one?

Reply:
Not sure if I got it right, but in the helm chart it is given as a [parameter](https://github.com/open-policy-agent/gatekeeper/blob/1076798df31db7c58cfd9def5f4972325fb519b8/charts/gatekeeper/templates/gatekeeper-manager-role-clusterrole.yaml#L32); and at least helm chart looks fine regarding the rbac side. 

By the way we've manipulated `mutatingWebhookName` in the values yaml, upgraded helm deployment and added `--mutatingWebhookName` flag manually in the controller; now everything work as expected.

Reply:
Oh perfect, apologies for missing that earlier!